tcms/telemetry/api/telemetry.py

```diff
@@ -0,0 +1,61 @@
+from django.db.models import Count
+from modernrpc.core import rpc_method
+
+from tcms.management.models import Priority
+from tcms.testcases.models import TestCase, Category, TestCaseStatus
+
+
+@rpc_method(name='Testing.breakdown')
+def breakdown(query=None):
+    """
+    .. function:: XML-RPC Testing.breakdown(query)
+
+        Perform a search and return the statistics for the selected test cases
+
+        :param query: Field lookups for :class:`tcms.testcases.models.TestCase`
+        :type query: dict
+        :return: Object, containing the statistics for the selected test cases
+        :rtype: dict
+    """
+
+    if query is None:
+        query = {}
+
+    test_cases = TestCase.objects.filter(**query)
+
+    count = {
+        'manual': test_cases.filter(is_automated=False).count(),
+        'automated': test_cases.filter(is_automated=True).count(),
+        'all': test_cases.count()
```

Comment:
Save the values in 2 variables and append them here. This will save a 3rd DB query.